test/unit/user_test.rb

```diff
@@ -110,7 +110,8 @@ class UserTest < ActiveSupport::TestCase
 
   test 'contributor_count' do
     contributor_count = User.contributor_count_for(Time.now-5.years, Time.now)
-    assert_equal contributor_count,4
+    is_positive = contributor_count>=0
+    assert is_positive
```

Comment:
You could try `assert_equal contributor_count, 0` and then read the test output to find what the correct value ought to be, then modify the test to match that.

Reply:
Although we may want to be careful -- any time someone adds more test data, this value could change, no? That could make adding new test data a bit of a pain.

Reply:
Yep, I guess that's why I got a failed test when I kept it equal to 4. It worked perfectly on my pc which contains a bit of manipulated data (got 4 using the method you told) but fails on travis. I guess this should me kept more generic and not equal to a numerical value. What do you think??

Reply:
Sure, we could just leave it at not 0 for now, i guess. Hmm. Doing more risks being circular, you know?

Reply:
We could test that it increments by 1 though, when adding a new comment... what do you think?

Reply:
Like here: https://github.com/publiclab/plots2/blob/master/test/unit/comment_mailer_test.rb#L7

Reply:
That's a good idea!! I'll have a look at it!